src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -293,7 +307,7 @@ describe('Detailed view and edit a selected work item - ', () => {
     comp.loggedIn = fakeAuthService.isLoggedIn();
```

Comment:
The comp.workItem is showing a type error.
TS2322:Type 'WorkItem' is not assignable to type 'WorkItem'.